### ISSUE TITLE
Reproduce performance of YOLOv9

### ISSUE THREAD

**Issue by johannes-tum** [WongKinYiu/YOLO#163]
Hi, 

to what extent could you already reproduce the results of the official YOLOv9 (and YOLOv7) repository?

Best wishes

Johannes

**Comment by tloki** [comment]
In my experience, Yolo-V9-S with coco trained weights (and used on subset of classes - 23 of them) exceeds accuracy of original implementation of yolov9 repo (also stock coco weights and 23 classes).

I tested usin pycocotools.

**Comment by johannes-tum** [comment]
That sounds great. Have you also tried larger models like the c model? Did you make any changes to the repository? Did you use the COCO pretrained weights from the official yolov9 repo? And what was the dataset you finetuned / tested on?
